src/cmap/auth/mongodb_oidc/aws_device_workflow.ts

```diff
@@ -0,0 +1,36 @@
+import { readFile } from 'node:fs';
```

Comment:
Similar to NODE-5048, you can do this but some bundlers will choke on it by default, we can work around it but it might save us some trouble if the driver doesn’t use the `node:` prefix for now :)

Reply:
Updated.